gef.py

```diff
@@ -229,12 +229,16 @@ def highlight_text(text: str) -> str:
     return "".join(ansiSplit)
 
 
-def gef_print(x: str = "", *args: Any, **kwargs: Any) -> Optional[int]:
-    """Wrapper around `print()`, using string buffering feature."""
-    x = highlight_text(x)
+def gef_print(*args: str, end="\n", sep=" ", **kwargs: Any):
```

Comment:
To be consistent with the other function that don't return anything:

```suggestion
def gef_print(*args: str, end="\n", sep=" ", **kwargs: Any) -> None:
```